simp.json.template

```diff
@@ -63,78 +77,74 @@
+        // set simp user password
         "simp<enter><wait><wait>UserPassword<enter>",
         "<wait><wait>",
         "UserPassword<enter>",
-        "<wait><wait>",
-        "<wait><wait>",
+        "<wait5>",
         "{{user `new_password`}}<enter>",
-        "<wait><wait>",
-        "<wait><wait>",
+        "<wait5>",
         "{{user `new_password`}}<enter>",
-        "<wait><wait>",
-        "<wait><wait>",
+        "<wait5>",
         "exit<enter>",
         "<wait10>",
+        // set root user password
         "root<enter><wait><wait>RootPassword<enter>",
-        "<wait><wait>",
-        "<wait><wait>",
+        "<wait5><wait>",
         "RootPassword<enter>",
-        "<wait><wait>",
-        "<wait><wait>",
+        "<wait5>",
         "{{user `new_password`}}<enter>",
-        "<wait><wait>",
-        "<wait><wait>",
+        "<wait5>",
         "{{user `new_password`}}<enter>",
-        "<wait><wait>",
-        "<wait><wait>",
-# Bring up the NAT network
+        "<wait5>",
         "echo 'simp ALL=(ALL) NOPASSWD:ALL'  >> /etc/sudoers<enter>",
-        "<wait><wait>",
-        "<wait><wait>",
-        "sed -ie '/Defaults *requiretty/d' /etc/sudoers<enter>",
-        "<wait><wait>",
-        "<wait><wait>",
-        "sed -ie 's/ONBOOT=no/ONBOOT=yes/g' /etc/sysconfig/network-scripts/ifcfg-{{user `nat_interface`}}<enter>",
-        "<wait><wait>",
-        "<wait><wait>",
-        "sed -ie 's/DEFROUTE=yes/DEFROUTE=no/g' /etc/sysconfig/network-scripts/ifcfg-{{user `nat_interface`}}<enter>",
-        "<wait><wait>",
-        "<wait><wait>",
+        "<wait5>",
+        "sed -i -e '/Defaults *requiretty/d' /etc/sudoers<enter>",
+        "<wait5>",
+        // configure NAT interface
+        "sed -i -e 's/ONBOOT=no/ONBOOT=yes/g' -e 's/DEFROUTE=yes/DEFROUTE=no/g' /etc/sysconfig/network-scripts/ifcfg-{{user `nat_interface`}}<enter>",
+        "<wait5>",
+        //"grep NM_CONTROLLED /etc/sysconfig/network-scripts/ifcfg-{{user `nat_interface`}} || echo NM_CONTROLLED=no >> /etc/sysconfig/network-scripts/ifcfg-{{user `nat_interface`}}<enter>",
+        "<wait5>",
+        // bring up NAT interface
         "ifup {{user `nat_interface`}}<enter>",
-        "<wait><wait>",
-        "<wait><wait>",
+        "<wait5>",
         "exit<enter>"
        ],
       "ssh_username": "simp",
-#      "ssh_private_key_file": "{{user `ssh_key`}}",
       "ssh_password": "{{user `new_password`}}",
+      // "ssh_private_key_file": "{{user `ssh_key`}}",
       "shutdown_command": "sudo /sbin/shutdown -P now"
     }
   ],
+
+  // Set up the users for packer and vagrant and bootstrap the system
+  // --------------------------------------------------------------------------
   "provisioners": [
-###########################################
-# Set up the users for packer and vagrant #
-# and boot strap the system               #
-###########################################
+
+    // Copy scripts from the host to the virtual machine for use later on
```

Comment:
Why change artificial commenting scheme to use '//' instead of '#'?  Seems like an arbitrary style change.

Reply:
@lnemsick-simp JSON is effectively JavaScript, so if we use JavaScript-style comments we can use an editor's JavaScript syntax highlighting to understand both code and comments.